### DISCUSSION TITLE
Lost Password

### DISCUSSION THREAD

**Discussion by s3w47m88** [towfiqi/serpbear#159] [General]
I feel foolish asking this question, but I am new to Docker. I have forgotten the password, and I cannot locate the docker-compose.yaml/yml file to reset it. And there is no password reset. :)

**Comment by towfiqi** [comment]
There is no default password and docker-compose.yml file. 
You should set an environment variable called PASSWORD and set it to whatever you like and then use that password to log in. 
https://docs.serpbear.com/deployment/running-locally-with-docker
